### ISSUE TITLE
Plot tags go missing when theme is applied to plot assemble

### ISSUE THREAD

**Issue by clauswilke** [thomasp85/patchwork#57]
I noticed a curious effect today. Seems to affect only the theme element `plot.tag.position`. Different plot tags go missing for different theme choices.

``` r
library(ggplot2)
library(patchwork)

df <- data.frame(x = 1:3, y = 1:3)
p <- ggplot(df, aes(x, y)) + geom_point()
p1 <- p + labs(tag = "A")
p2 <- p + labs(tag = "B")
p3 <- p + labs(tag = "C")

(p1 + p2) / p3
```

![](https://i.imgur.com/kou9buw.png)

``` r

((p1 + p2) / p3) & (theme_classic() + theme(plot.tag.position = "left"))
```

![](https://i.imgur.com/I6IWtNz.png)

``` r

((p1 + p2) / p3) & (theme_classic() + theme(plot.tag.position = "right"))
```

![](https://i.imgur.com/HXfG8di.png)

``` r

((p1 + p2) / p3) & (theme_classic() + theme(plot.tag.position = c(0, .9)))
```

![](https://i.imgur.com/8OM8xd9.png)

Created on 2018-05-18 by the [reprex package](http://reprex.tidyverse.org) (v0.2.0).

**Comment by thomasp85** [comment]
related to #56 - basically any elements that span across the plotting area (which tags does with absolute placement) will silently disappear when nested... This is of course not viable and I'm actively seeking a solution, but it is not straightforward...

**Comment by thomasp85** [comment]
closed in lieu of #119
